java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -87,14 +89,24 @@ public static Map<String, String> parseInfoResponseToMap(String serverInfo) {
 
     public static GlideClientConfiguration.GlideClientConfigurationBuilder<?, ?>
             commonClientConfig() {
-        return GlideClientConfiguration.builder()
-                .address(NodeAddress.builder().port(STANDALONE_PORTS[0]).build());
+        var builder = GlideClientConfiguration.builder();
+        for (var host : STANDALONE_HOSTS) {
+            var parts = host.split(":");
+            builder.address(
+                    NodeAddress.builder().host(parts[0]).port(Integer.parseInt(parts[1])).build());
+        }
+        return builder.useTLS(TLS);
```

Comment:
only apply if the standalone host is provided by the user. 
otherwise, we shouldn't use the TLS flag.